automated-tests/src/main/java/com/hp/ov/sdk/network/EthernetNetworkResource.java

```diff
@@ -112,6 +117,30 @@ public int getAssociatedUplinkGroups(String resourceID) {
         return -1;
     }
 
+    public String patchEthernetNetwork(String id) {

```

Comment:
This method name is not required since this is a class for Ethernet Network. 
Please rename it to `patch`.

Reply:
Done